### PR TITLE
First version of PAS Explanatory Report added

### PR DESCRIPTION
**Pull request by mbeckerle**
This is in markdown.

Use new line for each new sentence style. Long sentences can have multiple lines, but each new sentence should start on a new line.

### CODE REVIEW

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+# PAS Explanatory Report for Open Grid Forum (OGF) Data Format Description Language (DFDL)
+
+## ISO/IEC JTC 1 Common Strategic Characteristics
+_PAS Originators/Submitters are invited to explicitly reference the JTC 1 common strategic characteristics
+(interoperability, portability, cultural and linguistic adaptability, and accessibility) when submitting their
+PAS Submitter application or any PAS for transposition._
+
+### ORGANIZATION CRITERIA (SD9 7.3)
+OGF was approved as a PAS Submitter effective TBD: Date TBD: ApplicationID (e.g., like JTC 1 N14327).
```
[the remark this answers is not in the document]
Comment (mbeckerle) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:9]: I have a copy of the OGF PAS application . The document is undated, and has no identification number. Nor is there any evidence of it being approved. I need an approval date and a document ID number.

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+demonstrated for several DFDL format descriptions (called 'schemas'). 
+
+_b) How easily can implementation take place without need of additional descriptions?_
+
+The specification is self-contained. 
+It references other documents which collectively provide everything required to implement the language. 
+
+_c) What proof exists for successful implementations (e.g. availability of test results for media standards)?_
+
+IBM DFDL has hundreds of customer deployments.
```
[the remark this answers is not in the document]
Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md]: IBM WebSphere Message Broker (2011 - 2017)
IBM Integration Bus (2013 - 2022)
IBM App Connect Enterprise (2018 onwards)
IBM z/Transaction Processing Facility

[the remark this answers is not in the document]
Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md]: Also IBM DFDL has both C and Java implementations

[the remark this answers is not in the document]
Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md]: Also https://cloud.google.com/blog/products/application-modernization/dfdl-processing-with-google-cloud

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+##### 7.4.2.2 Response to User Requirements
+_a) How and when were user requirements considered and utilized?_
+
+User requirements are solicited via the regular Project calls and the mailing lists, including
+feedback received by the general public on an ongoing basis. This feedback is directly utilized in
+drafting each subsequent revision of the Specification.
+
+_b) To what extent have users demonstrated satisfaction?_
+
+TBD: can IBM say something about customer volume (hundreds?)
```
[the remark this answers is not in the document]
Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:167]: Approx 1300 IBM customers potentially using IBM DFDL for mission-critical integration & automation applications.

**Comment by smhdfdl** [comment]
@mbeckerle I have added comments to the places I was tagged.

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+The Specification was ratified as a Grid Full Recommendation by the OGF in February of 2021. 
+
+_b) To what extent and for how long have products been implemented using the specification?_
+
+IBM DFDL (several different implementations), the ESA (European Space Agency) and the Apache 
+Daffodil Open-Source project all began well before the completion of the OGF standard, and 
+revised their implementations as the specification was refined on the way to becoming a full OGF Recommendation. 
+
+TBD: Do we need dates here?
```
[the remark this answers is not in the document]
Comment (mbeckerle) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:98]: @smhdfdl I don't see an earlier comment with the specific product names.

[the remark this answers is not in the document]
Comment (smhdfdl) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md]: https://github.com/OpenGridForum/DFDL/pull/41#discussion_r1196619986

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+
+_b) How easily can implementation take place without need of additional descriptions?_
+
+The specification is self-contained. 
+It references other documents which collectively provide everything required to implement the language. 
+
+_c) What proof exists for successful implementations (e.g. availability of test results for media standards)?_
+
+IBM has two independent DFDL implementations. One is for a product called App Connect Enterprise (ACE), the other for the z/TPF system.
+Both have hundreds of customer deployments.
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:60]: z/TPF only has about 30 (very large!) customers

Comment (mbeckerle) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:60]: I will change the wording to reflect combined number of hundreds of customers.

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+_a) How and when were user requirements considered and utilized?_
+
+User requirements are solicited via the regular Project calls and the mailing lists, including
+feedback received by the general public on an ongoing basis. This feedback is directly utilized in
+drafting each subsequent revision of the Specification.
+
+_b) To what extent have users demonstrated satisfaction?_
+
+TBD: can IBM say something about customer volume (hundreds?)
+TBD: what else can we say here?
```
[the remark this answers is not in the document]
Comment (mbeckerle) [reply ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md]: Alas no.

**Comment by mbeckerle** [comment]
Last changes are in.

@smhdfdl with an official approval from you on this review I will squash 6 commits to one coherent commit and merge back to the master branch and let others know to submit it.

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+### ORGANIZATION CRITERIA (SD9 7.3)
+OGF was approved as a PAS Submitter effective 2022-12-02; the application ID is N16164.  
+Since attaining PAS Submitter status there have been no significant changes to OGF at an
+organizational level. 
+Additionally, there have been no changes to the organization criteria as submitted in
+the PAS submitter application N16164.
+
+### DOCUMENT RELATED CRITERIA (SD9 7.4)
+
+The DFDL Specification provides a
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:17]: DFDL 1.0

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+schema allows any text or binary data to be read (or "parsed") from its native format and to
+be presented as an instance of an information set. (An information set is a logical 
+representation of the data contents, independent of the physical format. 
+For example, two records could be in different formats, because one has fixed-length fields 
+and the other uses delimiters, but they could contain exactly the same data, and would both be 
+represented by the same information set). 
+The same DFDL schema also allows data to be taken from an instance of an information set and 
+written out (or "serialized") to its native format.
+
+This specification addresses a specific challenge faced response to a need for interoperable APIs
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:28]: faced **in** response

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 9]
```diff
@@ -0,0 +1,353 @@
...
+_Within its scope the specification shall completely describe the functionality (in terms of interfaces,
+protocols, formats, etc.) necessary for an implementation of the submission. 
+If it is based on a product, it
+shall include all the functionality necessary to achieve the stated level of compatibility or interoperability
+in a product independent manner._
+
+##### 7.4.1.1 Completeness (M)
+_a) How well are all interfaces specified?_
+
+The DFDL Specification has multiple implementations including several different implementations by IBM
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:47]: DFDL 1.0
IBM **C**orporation

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 10]
```diff
@@ -0,0 +1,353 @@
...
+#### 7.4.3 Alignment
+_The specification should be aligned with existing JTC 1 standards or ongoing work and thus complement
+existing standards, architectures and style guides. 
+Any conflicts with existing standards, architectures
+and style guides should be made clear and justified._
+
+##### 7.4.3.1 Relationship to Existing Standards
+_a) What International Standards are closely related to the specification and how?_
+
+The DFDL Specification is layered on top of W3C [XML Schema](https://www.w3.org/TR/xmlschema-0/), and
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:274]: DFDL 1.0
XML Schema 1.0

File: ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md [hunk 11]
```diff
@@ -0,0 +1,353 @@
...
+
+##### 7.4.3.3 Substitution and Replacement
+_a) What needs exist, if any, to replace an existing International Standard? Rationale?_
+
+Not applicable.
+
+_b) What is the need and feasibility of using only a portion of the specification as an International
+Standard?_
+
+DFDL is a large standard; hence, it explicitly allows _conforming_ subsets, and identifies
```
Comment (smhdfdl) [inline ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md:323]: I'm not sure that's the right answer. The question is asking whether ISO could adopt just part of the DFDL spec. We don't want that. So while you can point out that DFDL allows conforming subsets, we want to recommend that the whole spec is adopted, and is not structured in a way that makes it amenable to adopting only a portion.

**Review by smhdfdl — CHANGES_REQUESTED** [review]

**Comment by mbeckerle** [comment]
@smhdfdl I believe I fixed these additional issues now.

**Pull request merged**